responders/src/UI/embc-responder/src/app/core/services/global-constants.ts

```diff
@@ -40,24 +40,39 @@ export const usDefaultObject = {
   name: 'United States of America'
 };
 export const zipCodePattern = '^([0-9]{5}-[0-9]{4}|[0-9]{5})$';
+export const securityQuestionAnswerPattern = '^[a-zA-Z0-9 ]+$';
 export const petsQuantityPattern = '^([1-9][0-9]{0,2})$';
 
 export const gender = [
   { name: 'Male', value: 'Male' },
   { name: 'Female', value: 'Female' },
   { name: 'X', value: 'X' }
 ];
-export const wizardProfileMessage =
-  'Please complete all sections of the Evacuee Profile prior to submitting.';
-export const wizardESSFileMessage =
-  'Please complete all sections of the ESS File prior to submitting.';
-export const lockedStepMessage =
-  'Please complete the ESS File prior to proceeding to the next steps.';
+
+export const wizardProfileMessage = {
+  title: 'Complete all steps',
+  text:
+    'Please complete all sections of the Evacuee Profile prior to submitting.'
+};
+export const evacueeProfileCreatedMessage = {
+  title: 'Evacuee Profile Saved',
+  text: 'Evacuee profile has been successfully created.'
+};
+export const wizardESSFileMessage = {
+  title: 'Complete all steps',
+  text: 'Please complete all sections of the ESS File prior to submitting.'
+};
+export const lockedStepMessage = {
```

Comment:
locked step message needs to be dynamic. Define different messages for different steps